book_source/developers_guide/Adding-an-Input-Converter.Rmd

```diff
@@ -1,29 +1,30 @@
-General Note: dates in the database should be datatime (preferably with timezone), and datetime passed around in PEcAn should be of type POSIXlt.
+# How to Add an Input Converter
 
-# Met Data
+## Meterological Data conversion
 
-The standard met data inputs should be of the form:
+In general, you will need to write a function to download the raw met data andone to convert it to the PEcAn standard.
 
-Converters from a raw to standard format go in `/modules/data.atmosphere/R`; converters from standard to model-specific go in `models/<mymodel>/R`.
+Downloading raw data function are named `download.<source>.R`. Example functions can be found within the PEcAn directory [`/modules/data.atmosphere/R`](https://github.com/PecanProject/pecan/tree/develop/modules/data.atmosphere/R). 
```

Comment:
With data.atmosphere, it's more than just that you'll find examples there. These functions HAVE to be in that package's namespace for met.process to find them. Developers need to know that explictly, hence the previous "converters go in" language.